app/controllers/api/nextv1/connection/bitflyer_connections_controller.rb

```diff
@@ -0,0 +1,57 @@
+require "uri"
+require "net/http"
+require "json"
+require "digest"
+require "base64"
+
+class Api::Nextv1::Connection::BitflyerConnectionsController < Api::Nextv1::Oauth2Controller
```

Comment:
is this new or moved?

Reply:
This is all moved.  The originals can be found here: [app/controllers/payment/connection/bitflyer_connections_controller.rb](https://github.com/brave-intl/publishers/blob/staging/app/controllers/payment/connection/bitflyer_connections_controller.rb)